Source/WebKit/WebProcess/GPU/graphics/RemoteRenderingBackendProxy.cpp

```diff
@@ -292,14 +292,14 @@ void RemoteRenderingBackendProxy::releaseRemoteImageBufferSet(RemoteImageBufferS
     send(Messages::RemoteRenderingBackend::ReleaseRemoteImageBufferSet(bufferSet.identifier()));
 }
 
-void RemoteRenderingBackendProxy::moveToSerializedBuffer(WebCore::RenderingResourceIdentifier identifier)
+void RemoteRenderingBackendProxy::moveToSerializedBuffer(RemoteSerializedImageBufferReference ref)
 {
-    send(Messages::RemoteRenderingBackend::MoveToSerializedBuffer(identifier));
```

Comment:
With the types, the current code here says:
- Destroy ImageBuffer with the name `identifier`
- Create SerializedImageBuffer carrying the contents of the ImageBuffer above